conftest.py

```diff
@@ -451,24 +451,39 @@ def dask_env_variables():
     return {k: v for k, v in os.environ.items() if k.startswith("DASK_")}
 
 
-@pytest.fixture(scope="module")
-def small_cluster(request, dask_env_variables, gitlab_cluster_tags):
-    # Extract `backend_options` for cluster from `backend_options` markers
-    backend_options = merge(
-        m.kwargs for m in request.node.iter_markers(name="backend_options")
-    )
-    backend_options["send_prometheus_metrics"] = True
```

Comment:
This could only work at module level. I could not find it used anywhere and I wonder if there's any actual benefit in retaining the extra complexity? @jrbourbeau

Reply:
This may very well be dead code. If you don't see it being used anywhere, feel free to remove